sqlmesh/core/audit/definition.py

```diff
@@ -432,24 +358,109 @@ def text_diff(self, other: Node) -> str:
                 f"Cannot diff audit '{self.name} against a non-audit node '{other.name}'"
             )
 
-        return d.text_diff(self.query, other.query, self.dialect)
+        meta_a, *statements_a = self.render_definition()
+        meta_b, *statements_b = other.render_definition()
+
+        breakpoint()
```

Comment:
If the build didn't fail here, then our test coverage is lacking

Reply:
Added test